scheduler-run-jobs/fsm.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2016-2018 Workiva Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+machines:
+
+- name: scheduler-run-jobs
+
+  states:
+
+  - name: jobrun
+    initial: true
+    do_action: scheduler-run-jobs.actions.RunJobAction
+    transitions:
+    - target: jobdone
+      event: jobsuccess
+    - target: jobretry
+      event: jobfail
+
+  - name: jobdone
+    do_action: scheduler-run-jobs.actions.DoneJobAction
+    final: true
+
+  - name: jobretry
```

Comment:
I don't think retries need to be explicit in the state machine, since aws-lambda-fsm has logic for handling them.